### ISSUE TITLE
Allow tagging of channels with CPU architecture (x86 vs x64)

### ISSUE THREAD

**Issue by waltdestler** [itchio/butler#125]
I noticed that the [butler documentation](https://itch.io/docs/butler/pushing.html) says "Limits: there is no way to tag channels with architecture (32-bit vs 64-bit) yet." (I looked for a Github issue for this but didn't see one, so apologies if it already exists or I'm posting in the wrong place.)

The inability to upload different builds for 32-bit and 64-bit OS's is an issue blocking my game from using Butler. My game has separate 32-bit and 64-bit builds because the 64-bit version has better performance (and will probably be the only version to support multiplayer), but I still want to support those players running 32-bit OS's. (Sadly still pretty common these days.)

Currently I'm using Inno Setup to handle installation of the correct files, so this isn't a critical issue for me, but I know you'd prefer people to use Butler, so I figured I should let you know that this issue is blocking me from switching to Butler.

**Comment by fasterthanlime** [comment]
So, the docs are right in the sense that there's no specific `architecture` tag in the backend - so it's not really a butler limitation, more like an itch.io limitation.

However, whether or not you use butler, the itch.io app will attempt to pick the right upload based on the upload's name: https://github.com/itchio/itch/blob/000107e772d24da2c2e17b20d2b4d4429d4dbb22/src/reactors/downloads/find-uploads.ts#L170 - so if you use butler, and you upload both `windows-32` and `windows-64`, it should do the right thing still.

The plan for the future is for `32` and `64` in the channel name to set an `architecture` column in the database. Right now it's string-based, which is not as solid, but still better than nothing.

To summarize, nothing's stopping you from using butler right now - and you're right, I'd prefer if you use it over InnoSetup!

**Comment by waltdestler** [comment]
Ah, good to know, thanks!
